.github/workflows/build.yml

```diff
@@ -0,0 +1,31 @@
+name: build
+on:
+  push:
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ '8', '11', '14', '15-ea' ]
+        architecture: [ 'x64' ]
+    name: Build with JDK ${{ matrix.java }} on ${{ matrix.architecture }}
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
use checkout@v2

Reply:
Dependabot will prepare a PR for us as soon as we merge it :) but we could do that already. Thanks